graphql_compiler/tests/test_data_tools/schema.sql

```diff
@@ -64,6 +64,8 @@ CREATE INDEX Animal.net_worth NOTUNIQUE
 CREATE CLASS Animal_ParentOf EXTENDS E
 CREATE PROPERTY Animal_ParentOf.in LINK Animal
 CREATE PROPERTY Animal_ParentOf.out LINK Animal
+ALTER CLASS Animal_ParentOf CUSTOM human_name_in="Parent"
```

Comment:
Add a test on the generated schema graph that these properties exist and have their expected values?